### PR TITLE
[ADD] saas_portal.support_team model and notification

### PR DESCRIPTION
**Pull request by iledarn**
https://www.it-projects.info/web#id=287&view_type=form&model=project.task&action=152&active_id=29

### CODE REVIEW

File: saas_portal/data/subtype.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <!-- Client-related subtypes for messaging / Chatter -->
+        <record id="mt_expired" model="mail.message.subtype">
+            <field name="name">SaaS client's expiration date uccured</field>
```
Comment (yelizariev) [inline saas_portal/data/subtype.xml]:
```
Subscription expired
```

File: saas_portal/data/subtype.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            <field name="res_model">saas_portal.client</field>
+            <field name="default" eval="False"/>
+            <field name="description">SaaS client's expiration date uccured</field>
+        </record>
+        <!-- Support_team-related subtypes for messaging / Chatter -->
+        <record id="mt_support_team_expired" model="mail.message.subtype">
+            <field name="name">SaaS client's expiration date uccured</field>
```
Comment (yelizariev) [inline saas_portal/data/subtype.xml]:
```
Subscription expired
```

File: saas_portal/models/saas_portal.py [hunk 3]
```diff
@@ -430,13 +434,51 @@ class SaasPortalClient(models.Model):
     name = fields.Char(required=True)
     partner_id = fields.Many2one('res.partner', string='Partner', track_visibility='onchange')
     plan_id = fields.Many2one('saas_portal.plan', string='Plan', track_visibility='onchange')
-    expired = fields.Boolean('Expiration', compute='_get_expired')
+    expired = fields.Boolean('Expiration', default=False, readonly=True)
     user_id = fields.Many2one('res.users', default=lambda self: self.env.user, string='Salesperson')
+    trial = fields.Boolean('Trial', help='indication of trial clients', default=False, readonly=True)
+    notification_sent = fields.Boolean(default=False, readonly=True, help='notification about expiration sent')
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py]: notification about expiration was sent

File: saas_portal/models/wizard.py [hunk 4]
```diff
@@ -129,6 +129,7 @@ def _default_name(self):
     partner_id = fields.Many2one('res.partner', string='Partner')
     user_id = fields.Many2one('res.users', string='User')
     notify_user = fields.Boolean(help='Notify user by email when database will have been created', default=False)
+    support_team_id = fields.Many2one('saas_portal.support_team', 'Support Team')
```
Comment (yelizariev) [inline saas_portal/models/wizard.py]: add default support team (in res.users 's form

Comment (yelizariev) [inline saas_portal/models/wizard.py]: plus create support team

**Comment by iledarn** [comment]
исправил

**Comment by iledarn** [comment]
исправил автосопоставление инвойса и клиента

File: saas_portal/data/ir_config_parameter.xml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```
Comment (saltonmassally) [inline saas_portal/data/ir_config_parameter.xml:3]: noupdate=1 so as not to override changes that might be done via UI

Comment (yelizariev) [inline saas_portal/data/ir_config_parameter.xml:3]: sure, thank you

**Pull request merged**